web_tests/conftest.py

```diff
@@ -0,0 +1,72 @@
+from selenium import webdriver
+import pytest
+
+from web_tests.pages.login_page import LogiPage
+from web_tests.pages.change_password_page import ChangePassword
+from web_tests.helpers.users import Users
+from web_tests.helpers.passwords import Passwords
+
+VALID_USERNAME = 'Admin'
+
+VALID_PASSWORD = 'admin123'
+INVALID_PASSWORD = 'admin124'
+NEW_PASSWORD = 'Qwerty!12'
+MINIMUM_NUMBERS_6 = '123456'
+NO_LETTER_PASSWORD = '1234567'
+
+
+@pytest.fixture()
+def driver():
+    driver = webdriver.Chrome()
+    driver.maximize_window()
+    driver.implicitly_wait(6)
```

Comment:
You don't need implicit wait as you use explicit

Reply:
forgot to delete, I'll fix it